src/app/company/company.component.ts

```diff
@@ -3,6 +3,7 @@ import { Router, RouterModule } from '@angular/router';
 import { Company } from '../interfaces/company';
 import { DataService } from '../services/data.service';
 import { CommonModule } from '@angular/common';
+import { Job } from '../interfaces/job';
```

Comment:
@AleenaTim @Ishiano This import can be removed since `Job` is not being used anywhere in this component.